File: 2-Semestre/Taller de Programación/Personal/Python-C#/Ejercicio 19.cs
/*
El sueldo neto de un vendedor se calcula como la suma de un sueldo básico más el 12% del monto total vendido.
Determinar el sueldo neto de un vendedor sabiendo que hizo tres ventas en el mes.

Las ventas son ingresadas por el usuario.
*/

using System;

namespace Ejercicio_19
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Este programa calcula el sueldo neto de un vendedor\n");

            int sueldo_base;
            double ventas = 0;

            do
            {
                Console.Write("Ingrese el sueldo base del vendedor: ");
                sueldo_base = Convert.ToInt32(Console.ReadLine());

                if (sueldo_base <= 0)
                {
                    Console.WriteLine("[!] El sueldo base no puede ser menor o igual a 0");
                }
            } while (sueldo_base <= 0);

            for (int i = 1; i <= 3; i++)
            {
                int venta;

                do
                {
                    Console.Write($"Ingrese el monto de la venta {i}: ");
                    venta = Convert.ToInt32(Console.ReadLine());

                    if (venta <= 0)
                    {
                        Console.WriteLine($"El monto de la venta {i} no puede ser menor o igual a 0");
                    }
                } while (venta <= 0);

                ventas += venta;
            }

            double sueldo_neto = sueldo_base + (ventas * 0.12);

            Console.WriteLine($"El sueldo neto del vender es de ${sueldo_neto}");
        }
    }
}

File: 1-Semestre/Desarrollo de algoritmos/Actividades/Diagramas de flujo-Python (Semana 5)/Ejercicio 19.py
'''
El sueldo neto de un vendedor se calcula como la suma de un sueldo básico más el 12% del monto total vendido.
Determinar el sueldo neto de un vendedor sabiendo que hizo tres ventas en el mes.

Las ventas son ingresadas por el usuario.
'''

print("Este programa calcula el sueldo neto de un vendedor.")
sueldo_base = int(input("Ingrese el sueldo base del vendedor: "))
while sueldo_base <= 0:
    print("[!] El sueldo base debe ser mayor a 0")
    sueldo_base = int(input("Ingrese el sueldo base del vendedor: "))

venta1 = int(input("Ingrese el monto de la venta 1: "))
venta2 = int(input("Ingrese el monto de la venta 2: "))
venta3 = int(input("Ingrese el monto de la venta 3: "))
while venta1 <= 0 or venta2 <= 0 or venta3 <= 0:
    print("[!] El monto de las ventas debe ser mayor a 0")
    venta1 = int(input("Ingrese el monto de la venta 1: "))
    venta2 = int(input("Ingrese el monto de la venta 2: "))
    venta3 = int(input("Ingrese el monto de la venta 3: "))

ventas = (venta1 + venta2 + venta3) * 0.12
sueldo_neto = sueldo_base + ventas
print(f"El sueldo neto del vendedor es: ${sueldo_neto}")